addons/website_event/tests/test_event_internals.py

```diff
@@ -56,18 +61,18 @@ def test_process_attendees_form(self):
         }])
 
         form_details = {
-            '1-name': 'Pixis',
-            '1-email': 'pixis@gmail.com',
-            '1-phone': '+32444444444',
-            '1-event_ticket_id': ticket_id_1.id,
-            '2-name': 'Geluchat',
-            '2-email': 'geluchat@gmail.com',
-            '2-phone': '+32777777777',
-            '2-event_ticket_id': ticket_id_2.id,
-            'question_answer-1-%s' % self.event_question_1.id: '5',
-            'question_answer-2-%s' % self.event_question_1.id: '9',
-            'question_answer-0-%s' % self.event_question_2.id: '7',
-            'question_answer-0-%s' % self.event_question_3.id: 'Free Text',
+            '1-name-%s' % self.event_identification_question_1.id: 'Pixis',
```

Comment:
As we have a heuristic to test that we pick the first value for multiple identification questions, could be interesting to have 2 phone questions for example, and make sure we populate the field with the first answer.

Would also be nice to include the company_name into this.